contrib/sirupsen/logrus/logrus.go

```diff
@@ -0,0 +1,32 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+// Package logrus provides a log/span correlation hook for the sirupsen/logrus package (https://github.com/sirupsen/logrus).
+package logrus
+
+import (
+	"github.com/sirupsen/logrus"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+// DDContextLogHook logs any span in the log context by implementing the logrus.Hook interface
```

Comment:
```suggestion
// DDContextLogHook ensures that any span in the log context is correlated to log output.
```
Is this not clearer? The feature is called "log correlation".